src/ingameop.cpp

```diff
@@ -224,15 +224,13 @@ static bool _intAddInGameOptions()
 
 	auto const &parent = psWScreen->psForm;
 
-	bool s = (bMultiPlayer && NetPlay.bComms != 0) || bInTutorial;
-
 	// add form
 	auto ingameOp = std::make_shared<IntFormAnimated>();
 	parent->attach(ingameOp);
 	ingameOp->id = INTINGAMEOP;
 	ingameOp->setCalcLayout(LAMBDA_CALCLAYOUT_SIMPLE({
-		bool s = (bMultiPlayer && NetPlay.bComms != 0) || bInTutorial;
-		psWidget->setGeometry(INTINGAMEOP_X, INTINGAMEOPAUTO_Y(s? 3 : 5), INTINGAMEOP_W, INTINGAMEOPAUTO_H(s? 3 : 5));
+		int lines = ((bMultiPlayer && NetPlay.bComms) || bInTutorial) ? 3 : 5;
```

Comment:
I think you can remove the parantheses around the condition, but this is more of a personal preference here.